docs/src/modules/getting-started/pages/planner-agent/dynamic-team.adoc

```diff
@@ -0,0 +1,165 @@
+= AI Planner Part 5: Dynamic orchestration
+
+include::ROOT:partial$include.adoc[]
+
+xref:planner-agent/index.adoc[1: The activity agent] > xref:planner-agent/preferences.adoc[2: User preferences] > xref:planner-agent/weather.adoc[3: Weather agent] > xref:planner-agent/team.adoc[4: Orchestrate the agents] > xref:planner-agent/list.adoc[5: List by user] > **6: Dynamic orchestration**
+
+include::ROOT:partial$new-to-akka-start-here.adoc[]
+
+== Overview
+
+We have used a workflow with predefined steps to call the `WeatherAgent` followed by the `ActivityAgent`. In a larger system there can be many agents, and it can be would be cumbersome to define a single workflow that would handle all types of requests. A more flexible approach is to let the AI model come up with a plan of which agents to use and in which order to achieve the goal of the request.
```

Comment:
```suggestion
We have used a workflow with predefined steps to call the `WeatherAgent` followed by the `ActivityAgent`. In a larger system there can be many agents, and it would be cumbersome to define a single workflow that would handle all types of requests. A more flexible approach is to let the AI model come up with a plan of which agents to use and in which order to achieve the goal of the request.
```